### ISSUE TITLE
delay "skip forward"

### ISSUE THREAD

**Issue by Thornburgh** [clsid2/mpc-hc#3015]
I just updated from a 2022 version of mpc-hc to the latest (Aug 2024) and immediately noticed a delay when I hit "skip forward" to the next video in my playlist.  It doesn't always happen, but it happens a lot, and sometimes I even get a CTD during the delay.  Reverting to the 2022 version fixed it.

**Comment by clsid2** [comment]
Disabling seek preview will probably fix that.

I am unable to reproduce any issue. It only affects small group of people and likely is caused by an external factor such as a graphics driver bug.

**Comment by Thornburgh** [comment]
Thank you.  I reinstalled and also turned off seek preview, and it now works correctly without delay.  I'm surprised seek preview is related, however, since I'm skipping to the next file by pressing a key and seek preview is not visible.

**Comment by clsid2** [comment]
The delay happens when stopping the filter graph that is used by preview. It is irrelevant if preview was shown or not.

I assume you have Intel GPU? The others who had this problem all did.

**Comment by Thornburgh** [comment]
Thanks.  This is actually a recent high-end computer with an Nvidia 3800.  But I'm using the x86 version of mpc-hc along with a very old ffdshow raw video filter to control scaling under various conditions (I have a large collection of music videos and some are very old and low resolution, so I don't want them to be scaled up too much).

**Comment by clsid2** [comment]
And a CPU without an integrated GPU?

ffdshow has x64 version. Not sure why you need a super old version.

You can also limit scaling within MPC-HC itself.
Menu > View > Video Frame > Normal Size (or double)
Assuming you want to watch in fullscreen.
Don't forget to return setting to default "Touch window from inside" for normal movie viewing.
Or make a copy of player and store settings in INI. Use that for the music.

**Comment by Thornburgh** [comment]
Yes, it's an AMD 5800x3D cpu with no integrated graphics.  

And thanks for the suggestions.  Years ago I concluded ffdshow x86 worked better and have never bothered to change it, but maybe I should.  (I've been using mpc-hc with ffdshow for almost 20 years.)  I actually use "touch window from outside" to scale music videos by default to fill the screen on a large TV, even if they were originally 4:3.  Then I use ffdshow to override that behavior under certain conditions (e.g., if the video is less than 640x480 or if I've added "nz" to the filename).
